.travis.yml

```diff
@@ -0,0 +1,14 @@
+notifications:
+    slack: wtsi-cgpit:ABlfGMqViryJ0JIhJuaWeXDa
+
+sudo: false
+
+services:
+    - docker
+
+script:
+- set -e
+- echo 'Build and check docker image'
+- docker build -t sigprofiler-travis .
+- docker images | grep -c sigprofiler-travis
+- docker run -t --rm sigprofiler-travis docker-sigprofiler -h
```

Comment:
We generally test that a version is output rather than dumping all of the help.  The ability to emit the version in a succinct way is important.

Reply:
> tag the repository in the settings with the licence

Do you know the keyword for AGPL3.0 license? I did not manage to make the toolbar display license info. The license has been confirmed this morning.

Reply:
Settings -> Insights -> Community -> Licence "Add"

Select: `GNU Affero General Public License Version 3`